clang/runtime/dpct-rt/include/dpl_extras/dpcpp_extensions.h.inc

```diff
@@ -80,7 +80,7 @@ constexpr auto __inclusive_scan_over_group(_Args... __args) {
 /// perform the scan.
 template <typename Item, typename T, class BinaryOperation,
           int VALUES_PER_THREAD>
-__attribute__((always_inline)) void
+__dpct_inline__ void
```

Comment:
`__dpct_inline` defined in `dpct/dpct.hpp`, so ,I think we should include `dpct/dpct.hpp` in this file, WDYT?

Reply:
updated to include `../dpct.hpp`